Point-of-No-Return/Scene.cpp

```diff
@@ -5,63 +5,47 @@
 #include "Help.h"
 #include "Ending.h"
 
-void SceneManager::MakeSnece(SceneID sceneID)
+void SceneManager::CreateScene(SceneID sceneID)
 {
+	SceneManager::sceneBase = nullptr;
 	//シーン分け
 	switch (sceneID)
 	{
 	case SceneManager::TitleID:
-		SceneManager::basescene = new Title();
+		SceneManager::sceneBase = new Title();
 		break;
 	case SceneManager::HelpID:
-		SceneManager::basescene = new Help();
+		SceneManager::sceneBase = new Help();
 		break;
 	case SceneManager::GameID:
-		SceneManager::basescene = new Game();
+		SceneManager::sceneBase = new Game();
 		break;
 	case SceneManager::EndingID:
-		SceneManager::basescene = new Ending();
+		SceneManager::sceneBase = new Ending();
 		break;
 	}
 }
 
-void SceneManager::Initialize(SceneID sceneID_)
+void SceneManager::Initialize(SceneID scene_id)
 {
-	sceneID = sceneID_;
-	save_sceneID = sceneID_;
-	SceneManager::MakeSnece(sceneID_);
+	SceneManager::CreateScene(scene_id);
 }
 
-void SceneManager::ChangeScene()
+void SceneManager::ChangeScene(SceneID scene_id)
 {
-	if (SceneManager::Changed())
-	{
-		SceneManager::DeleteScene();
-		SceneManager::MakeSnece(SceneManager::sceneID);
-	}
-	SetSaveScene();
+	SceneManager::DeleteScene();
+	SceneManager::CreateScene(scene_id);
 }
 
-bool SceneManager::Changed()
+void SceneBase::Load()
 {
-	if (sceneID != save_sceneID)
-	{
-			return true;
-	}
-	return false;
-}
-
 
+	HANDLE th = nullptr;
```

Comment:
`th` は直後に `CreateLoadThread()` の戻り値で初期化されるので、全く意味の無いコードですね 💩 

```suggestion
	HANDLE th = CreateLoadThread();
```
で良いのでは？

Reply:
確かにnullptrで初期化する必要ないですね。
修正しておきます。